src/Symfony/Component/Workflow/Tests/WorkflowTest.php

```diff
@@ -195,6 +195,34 @@ public function testBuildTransitionBlockerListReturnsUndefinedTransition()
         $workflow->buildTransitionBlockerList($subject, '404 Not Found');
     }
 
+    public function testBuildTransitionBlockerList()
+    {
+        $definition = $this->createComplexWorkflowDefinition();
+        $subject = new \stdClass();
+        $subject->marking = null;
+        $workflow = new Workflow($definition, new MultipleStateMarkingStore());
+
+        $this->assertTrue($workflow->buildTransitionBlockerList($subject, 't1')->isEmpty());
+        $this->assertFalse($workflow->buildTransitionBlockerList($subject, 't2')->isEmpty());
+
+        $subject->marking = array('b' => 1);
+
+        $this->assertFalse($workflow->buildTransitionBlockerList($subject, 't1')->isEmpty());
+        // In a workflow net, all "from" places should contain a token to enable
+        // the transition.
```

Comment:
How is this comment related to the code being executed (maybe I just misunderstand what this tries to explain)?

Reply:
if you want to, you can update that test too

Reply:
I'll let it be.